html/modules/custom/docstore/src/ManageFields.php

```diff
@@ -95,6 +95,8 @@ public function allowedFieldTypes() {
       'string_long' => 'text',
       'geofield' => 'string',
       'link' => 'string',
+      'telephone' => 'string',
+      'address' => '',
```

Comment:
just checking that this mapping is okay.

Reply:
It is, it avoids adding the field to the index